actionpack/test/controller/caching_test.rb

```diff
@@ -406,4 +419,25 @@ def test_caching_works_with_beginning_comment
     ActionView::PartialRenderer.expects(:collection_with_template).never
     get :index_with_comment
   end
+
+  def test_caching_works_with_collection_rendering_multiple_templates
+    get :index_with_multiple_templates
+
+    ActionView::PartialRenderer.expects(:collection_without_template).never
```

Comment:
It's definitely confusing. Especially because it renders a collection with many templates (I guess if you think of it as rendering a collection without a fixed template it makes sense).

--
Kasper

> Den 01/04/2015 kl. 05.35 skrev Rafael Mendonça França <notifications@github.com>:
> 
> In actionpack/test/controller/caching_test.rb:
> 
> > @@ -406,4 +419,25 @@ def test_caching_works_with_beginning_comment
> >      ActionView::PartialRenderer.expects(:collection_with_template).never
> >      get :index_with_comment
> >    end
> > +
> > +  def test_caching_works_with_collection_rendering_multiple_templates
> > +    get :index_with_multiple_templates
> > +
> > +    ActionView::PartialRenderer.expects(:collection_without_template).never
> Ah, so my font of confusion is the name of the method. For me collection_without_template is not say what it means, that is render the template for the collection. Maybe we should rename it to render_collection_without_template?
> 
> —
> Reply to this email directly or view it on GitHub.
>